src/util.c

```diff
@@ -10,12 +10,13 @@
 #include <strings.h>
 #include <stdlib.h>
 #include <ctype.h>
-#include <assert.h>
+
 #include "redisraft.h"
 
 int RedisModuleStringToInt(RedisModuleString *str, int *value)
```

Comment:
Maybe for another PR - but do we want to keep the `RedisModules` prefix for internal lib functions?

Reply:
This can be part of another PR. Ideally, I'd like to see namespace usage per file, like util functions should start with `Util...`.